docker/Dockerfile.ubuntu-20

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:focal
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+ARG SKIP_BUILD=
+ARG ZEEK_LTS=1
+ARG ZEEK_VERSION=3.0.11-0
+
+CMD ["sh"]
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends curl ca-certificates gnupg2 \
+ # Zeek.
+ && mkdir -p /tmp/zeek-packages \
+ && cd /tmp/zeek-packages \
+ && if [ -n "${ZEEK_LTS}" ]; then ZEEK_LTS="-lts"; fi && export ZEEK_LTS \
+ && curl -L --remote-name-all \
+    https://download.zeek.org/binary-packages/xUbuntu_20.04/amd64/zeek${ZEEK_LTS}_${ZEEK_VERSION}_amd64.deb \
+    https://download.zeek.org/binary-packages/xUbuntu_20.04/amd64/zeek${ZEEK_LTS}-core_${ZEEK_VERSION}_amd64.deb \
+    https://download.zeek.org/binary-packages/xUbuntu_20.04/amd64/zeekctl${ZEEK_LTS}_${ZEEK_VERSION}_amd64.deb \
+    https://download.zeek.org/binary-packages/xUbuntu_20.04/amd64/zeek${ZEEK_LTS}-core-dev_${ZEEK_VERSION}_amd64.deb \
+    https://download.zeek.org/binary-packages/xUbuntu_20.04/amd64/libbroker${ZEEK_LTS}-dev_${ZEEK_VERSION}_amd64.deb \
+    https://download.zeek.org/binary-packages/xUbuntu_20.04/amd64/zeek${ZEEK_LTS}-libcaf-dev_${ZEEK_VERSION}_amd64.deb \
+ && apt-get install -y libpcap0.8 libpcap-dev python libssl-dev zlib1g-dev libmaxminddb0 libmaxminddb-dev \
+ && dpkg -i *.deb \
+ && cd - \
+ && rm -rf /tmp/zeek-packages \
+ # Spicy build and test dependencies.
+ && apt-get install -y --no-install-recommends git ninja-build ccache bison flex libfl-dev python3 python3-pip zlib1g-dev jq locales-all python3-setuptools python3-wheel make \
+ && pip3 install btest==0.64 pre-commit \
+ # Clang-9.
+ && apt-get install -y --no-install-recommends llvm-9-dev clang-9 libclang-9-dev libc++-dev libc++1 libc++abi-dev libc++abi1 \
```

Comment:
Another question that applies to all Dockefiles: any newer LLVM/clang available?

Now that 11 is out, I think we should move away from 9. (I'd say lets just make sure that we always have the two most recent versions are in CI)

Reply:
In this case we use the stock distribution packages for LLVM since it seemed slightly nicer to users. If we want to across the board require a newer version, some distributions might need to build their own LLVM or get them from the LLVM repos.

In the current situation I think we should prefer distribution packages.